docs/reference/api/media.md

```diff
@@ -0,0 +1,153 @@
+# Media APIs
+
+
+The `@apostrophecms/attachment` module manages all media file upload. Once uploaded, metadata and organization of image and other files are primarily managed through the respective `@apostrophecms/image` and `@apostrophecms/file` piece types. See the [pieces REST API documentation](./pieces.md) for additional information.
```

Comment:
Adding attachment fields directly to your own piece types is a real thing that should be mentioned. The classic example is a headshot for a "person" piece: you don't want to clutter up the image library with it.